docs/BUILDING.md

```diff
@@ -0,0 +1,39 @@
+Whenever Rez releases a new versioned tag, this documentation is automatically built.
+
+TODO : Replace this URL once there's a real URL, later
+You can see the documentation here: https://rez-rtd-test.readthedocs.io/en/latest
+
+However, if you'd like to build the documentation locally, you can do so using
+these steps:
+
+- Install virtualenv
+
+```sh
+python3 -m pip install virtualenv --user
+```
+
+- Create a virtual environment and source it
+
+```sh
+virtualenv rez_documentation_environment
+source rez_documentation_environment/bin/activate
+```
+
+- Install the pip requirements
+
+```sh
+python -m pip install --requirement requirements.txt
+```
+
+- Now build the documentation
+
+```sh
+sphinx-build . _build
```

Comment:
I'd recommend users to use `make html` or the likes instead of `sphinx-build`. But I think the makefiles are missing, so a new bootstrap wuld be welcome.